config/initializers/inflections.rb

```diff
@@ -11,6 +11,6 @@
 # end
 
 # These inflection rules are supported but not enabled by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
-# end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym 'API'
```

Comment:
@littleforest 

I forgot to ask last review -- what does this do? I'm not seeing mention of it on the gem's readme.

Reply:
It allows us to have modules/classes with "API" instead of "Api".  Ordinarily `app/controllers/api/v1/occupations_controller.rb` would require the class name to be `Api::V1::OccupationsController`, but after setting the inflection acronym, we have `API::V1::OccupationsController` instead.

Reply:
Ah. I didn't even notice this was a rails api thing. I've never run into this. Thanks!